src/shared/components/query/QueryStore.ts

```diff
@@ -1556,14 +1565,16 @@ export class QueryStore {
     getFilteredProfiles(
         molecularAlterationType: MolecularProfile['molecularAlterationType']
     ) {
-        return this.molecularProfilesInSelectedStudies.result.filter(
+        const ret = this.molecularProfilesInSelectedStudies.result.filter(
```

Comment:
Do we really need to declare a new const `ret`? We just return `ret` without doing anything else, right?